web_source/_quarto.yml

```diff
@@ -18,44 +18,49 @@ book:
   repo-actions: [edit, issue]
   chapters:
     - index.qmd
-    - acknowledgements.qmd
-    - acronyms.qmd
-    - cite.qmd
-    - disclaimer.qmd
+    - pref-acknowledgements.qmd
+    - pref-acronyms.qmd
+    - pref-cite.qmd
+    - pref-disclaimer.qmd
     
-    - part: echild-overview.qmd
+    - part: echild.qmd
       chapters:
-        - intro.qmd
-        - overview-general.qmd
-        - overview-structure.qmd
+        - echild-introduction.qmd
+        - echild-information-scope.qmd
+        - echild-key-research-purposes.qmd
+        - echild-overview.qmd
+        - echild-database-structure.qmd
         
-    - part: echild-data.qmd
+    - part: data.qmd
       chapters:
+        - data-attribute-data.qmd
         - data-coverage.qmd
         - data-hes.qmd
-        - data-other.qmd
         - data-npd.qmd
         - data-mother-baby.qmd
 
-    - part: echild-access.qmd
+    - part: access.qmd
       chapters:
-        - access-how.qmd
+        - access-five-safes.qmd
+        - access-no-other-linkage.qmd
+        - access-ethics.qmd
+        - access-access-conditions.qmd
+        - access-process.qmd
+        - access-accredited-researcher.qmd
+        - access-aoc.qmd
         
-    - part: echild-quality.qmd
+    - part: sl.qmd
```

Comment:
Maybe expand filename, same reasons as above.

Reply:
Have renamed all, though is it possible to have subfolders to help with organisation?

Reply:
It would be possible but a possibly a little bit hacky, essentially I could create a script that moved files in subfolders (except “special” subfolders, like “images”) into the root folder before compiling/rendering.